osh/hub/scan/models.py

```diff
@@ -1283,3 +1283,26 @@ def csmock_args(self):
             # there are no arguments, this is not an error
             logger.info("No csmock arguments for profile '%s'", self)
             return ''
+
+
+class RetentionPolicySettingsManager(models.Manager):
+    def get_days_from_category(self, category):
+
+        # We establish a broad default days, so we don't delete unwanted results.z
+        try:
+            days = self.get(category=category).days
+        except ObjectDoesNotExist:
+            logger.warning("Category %s could not be found. Setting default parameter", category)
+            days = 60
```

Comment:
Previous iteration had `14` as the default number of days for an unknown category.  Is the change to `60` expected?

Reply:
Sure, makes perfect sense.  I just wanted to be sure it is not a typo.  Thank you!